Emby.Server.Implementations/Data/SqliteItemRepository.cs

```diff
@@ -6067,6 +6068,13 @@ private MediaStream GetMediaStream(IReadOnlyList<IResultSetValue> reader)
             if (reader[4].SQLiteType != SQLiteType.Null)
             {
                 item.Language = reader[4].ToString();
+
+                string i18nKey = "Language" + StringHelper.FirstToUpper(item.Language);
```

Comment:
```suggestion
                string i18nKey = "Language" + item.Language;
```

The localized string dictionary is case insensitive so this isn't needed